docs/src/main/resources/content/replication-configuring.adoc

```diff
@@ -104,4 +104,15 @@ of the replication configuration regardless of the results of the above options.
 === Viewing Replication Configurations
 
 To view a list of all currently saved replication configurations, simply execute `replication:config-list`
-in the ${command-console}.
\ No newline at end of file
+in the ${command-console}.
+
+=== Setting Timeouts
+
+Here's how you can set the timeouts for requests to a ${branding}:
```

Comment:
❓ Should we include a why? For example, `If your network is slow, increasing the client timeouts for replication can be increased to allow more time for transferring resources.`